src/Select/style.css

```diff
@@ -0,0 +1,119 @@
+@import '../assets/style/form-item.css';
```

Comment:
This style is only needed in the story. It should not be part of the component. So it can be imported in the story.ts file